modular_skyrat/modules/goofsec/code/projectiles.dm

```diff
@@ -201,3 +201,279 @@
 	damage = 60
 */
 
+/// Bullet security pistol.
+/obj/item/gun/ballistic/automatic/pistol/security
+	name = "security pistol"
+	desc = "A pistol issued to station security personnel. It's a small, compact pistol, but it packs a punch."
+	mag_type = /obj/item/ammo_box/magazine/security_pistol
+	icon = 'modular_skyrat/modules/sec_haul/icons/guns/ladon.dmi'
+	righthand_file = 'modular_skyrat/modules/sec_haul/icons/guns/inhands/righthand40x32.dmi'
+	lefthand_file = 'modular_skyrat/modules/sec_haul/icons/guns/inhands/lefthand40x32.dmi'
+	icon_state = "ladon"
+	inhand_icon_state = "ladon"
+	can_suppress = FALSE
+	fire_sound = 'modular_skyrat/modules/sec_haul/sound/pistol_fire.ogg'
+	rack_sound = 'sound/weapons/gun/pistol/rack.ogg'
+	lock_back_sound = 'sound/weapons/gun/pistol/slide_lock.ogg'
+	bolt_drop_sound = 'sound/weapons/gun/pistol/slide_drop.ogg'
+
+/obj/item/ammo_box/magazine/security_pistol
+	name = "influence security pistol magazine"
+	desc = "A magazine for the security pistol. Loaded with rubberized influence rounds."
+	icon = 'modular_skyrat/modules/goofsec/icons/gun_sprites.dmi'
+	icon_state = "security_pistol_influence"
+	base_icon_state = "security_pistol_influence"
+	ammo_type = /obj/item/ammo_casing/security_pistol/influence
+	caliber = CALIBER_SECURITYPISTOL
+	max_ammo = 20
+
+/obj/item/ammo_box/magazine/security_pistol/update_icon_state()
+	. = ..()
+	icon_state = "[base_icon_state][ammo_count() ? "" : "-empty"]"
+
+/// Ammo for the Security Pistol.
+/obj/item/ammo_casing/security_pistol
+	name = "security pistol casing"
+	desc = "A casing for a security pistol magazine."
+	icon = 'modular_skyrat/modules/sec_haul/icons/guns/ammo_cartridges.dmi'
+	icon_state = "sl-casing"
+	caliber = CALIBER_SECURITYPISTOL
+	projectile_type = /obj/projectile/bullet/security_pistol
+
+/obj/projectile/bullet/security_pistol
+	name = "security pistol bullet"
+	damage = 0 // not supposed to exist
+
+// Influence rounds. Bouncy. Stamina only.
+/obj/item/ammo_box/magazine/security_pistol/influence
+	name = "influence security pistol magazine"
+	desc = "A magazine for the security pistol. Loaded with Influence rounds, which bounce around and influence people to comply with arrests."
+	icon_state = "security_pistol_influence"
+	base_icon_state = "security_pistol_influence"
+	ammo_type = /obj/item/ammo_casing/security_pistol/influence
+
+/obj/item/ammo_casing/security_pistol/influence
+	name = "influence security pistol casing"
+	projectile_type = /obj/projectile/bullet/security_pistol/influence
+	harmful = FALSE
+
+/obj/projectile/bullet/security_pistol/influence
```

Comment:
Don't give them the best non lethal cartridge to start with. Make it do a low amount of damage.

Reply:
reduced to 30